tools/boilerplate.py

```diff
@@ -304,7 +304,10 @@ def boilerplate_gen():
         'pcolormesh': 'sci(__ret)',
         'hist2d': 'sci(__ret[-1])',
         'imshow': 'sci(__ret)',
-        'spy': 'if isinstance(__ret, cm.ScalarMappable): sci(__ret)  # noqa',
+        'spy': (
+            'if isinstance(__ret, cm.ScalarMappable):\n'
+            '        sci(__ret)'
```

Comment:
just confirming that this looks over indented because the parent has a layer of indentation applied to it?

Reply:
Yes, the `if` gets inserted at a 1-indent level already. The real change here is removing the `noqa`, since we run `black` on these auto-generated lines, and so it's no longer an if+code on a single line.